src/xtgeo/grid3d/grid.py

```diff
@@ -629,7 +629,7 @@ def to_file(self, gfile, fformat="roff"):
         elif fformat == "egrid":
             _grid_export.export_egrid(self, gfile.name)
         else:
-            raise SystemExit("Invalid file format")
+            raise ValueError(f"Invalid file format: {fformat}")
```

Comment:
Include accepted formats?

Reply:
Found it a bit verbose, but it would be a good addition I suppose, added it now.

Reply:
Just do the values?

`", ".join(itertools.chain.from_iterable(valid_formats.values()))`

Reply:
Considered that as well, but did not for two reasons, it is quite involved to import `itertools` just for an error message, but the main reason is it now gives the user more information (for example the `roff` vs `roff_ascii` difference).